spec/factories/topics.rb

```diff
@@ -2,7 +2,9 @@
 
 FactoryBot.define do
   factory :topic do
-    name 'What is your secret superpower?'
+    sequence :name do |n|
+      "This is my topic number # #{"%.8d" % n}"
```

Comment:
This is a useful change, but not really part of this pull request.

Reply:
this change is because I could get false positives with my specs due to the way I was using sequences before and then generating a large number of topics (this was your original feedback :D - further up the page!) 

Happy to take it out though... ?

Reply:
😳 ok.